tests/wpt/meta/fetch/api/abort/general.any.js.ini

```diff
@@ -5,46 +5,16 @@
   expected: ERROR
 
 [general.any.html]
-  [response.arrayBuffer() rejects if already aborted]
-    expected: FAIL
-
-  [response.blob() rejects if already aborted]
-    expected: FAIL
-
-  [response.formData() rejects if already aborted]
-    expected: FAIL
-
-  [response.json() rejects if already aborted]
-    expected: FAIL
-
-  [response.text() rejects if already aborted]
-    expected: FAIL
-
   [Call text() twice on aborted response]
     expected: FAIL
 
-  [response.bytes() rejects if already aborted]
+  [Stream will not error if body is empty. It's closed with an empty queue before it errors.]
```

Comment:
We should figure out why this one fails now; I think it must relate to the changes in some way. 

The idea in [the test](implement-read-looping-using-a-read-request) is that the item returned by `read` should be `done`, and remain unaffected by the abort. 

Haven't been able to figure it out yet.